index.html

```diff
@@ -23,6 +23,12 @@
       margin: 1.5rem 3rem;
     }
 
+    .indicator {
+      background:
+        url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAB4AAAAeCAYAAAA7MK6iAAAAtElEQVRIx+3SoQrCUBSA4YFgEUTMVk2+hdHoY/gAYtkLTIPFaLZY9Om+ZjmWW6a4Ow33h8PYYexj41ZVqVT6hzD/CYpH7zj2Ae/7RGe4B3zHrA90il2gr9lh2jW0QI0zbgmYzi2eq7H4Fp7Eyx4fzBmTLr56jNOb6AnjLn/5CE0L2mCU42AtW+BlrhO9boHXueBtAl2T+20u+BjABSsM4nqJ/TEXfMAGw2Q/jP2hKpVKpdw9AcDkjaObIxshAAAAAElFTkSuQmCC) no-repeat center / 3rem;
```

Comment:
I couldn't see this in the default 'settiings' theme. Using `data-icon="star-full"` might work better :snowman: